Wordle.Api/Wordle.Api/Services/LeaderboardService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Data;
+
+namespace Wordle.Api.Services
+{
+    public class LeaderboardService
+    {
+        private readonly AppDbContext _db;
+
+        public LeaderboardService(AppDbContext db)
+        {
+            _db = db;
+        }
+
+
+        public async Task<IEnumerable<Player>> GetTopPlayers(int? count)
+        {
+            count ??= 10;
+            
+            var players = await _db.Players
+                .Where(player => player.GameCount > 10)
+                .OrderBy(p => p.AverageAttempts)//Lowest average attempts
+                .ThenByDescending(p => p.GameCount)//Then by highest number of games played
+                .Take(count.Value)
+                .ToListAsync();
+            return players;
+        }
+
+        public async Task<Player> AddScore(string? name, int numAttempts, int seconds)
+        {
+            if (name is null)
+            {
+                throw new ArgumentException("Passed name is null");
+            }
+            var player = await _db.Players.FirstOrDefaultAsync(p => p.Name == name);
+            if (player != null)
+            {
+                // update player data
+                //word.IsCommon = isCommon;
```

Comment:
Probably get rid of this. Just in general, clean up stuff if you won't use it

Reply:
I forgot to delete that when I gave up on the player.csv file.  I'll get it removed.